assembl/lib/rds_token_url.py

```diff
@@ -0,0 +1,70 @@
+from datetime import datetime, timedelta
+
+from sqlalchemy.engine.url import URL
+import boto3
+
+
+class RdsTokenUrl(URL):
+    """An URL that uses IAM connection tokens as passwords"""
+    def __init__(
+            self, drivername, region=None, username=None, host=None,
+            session_name=None, port=None, database=None, query=None):
+        super(RdsTokenUrl, self).__init__(
+            drivername, username=username, host=host, port=port or 5432,
+            database=database, query=query)
+        self.aws_region = region
+
+    def get_rds_client(self):
+        if not getattr(self, 'rds_client', None):
```

Comment:
Could be False as default value, since we are testing the existence of the attribute

Reply:
You could, won't change anything. But you're looking for a key in an object, basically. If the key isn't there, you either throw an exception, or return `None` to indicate nothing was found. It's falsy enough for the condition clause it's used in.